### ISSUE TITLE
Nested properties again: live not dead) 

### ISSUE THREAD

**Issue by dmitryLagoda** [SoftwareBrothers/admin-bro#244]
Really tough issue) 
Easy to check this time: just add a new Complicated resource in your example app. Nested fields don't save data.
```
    "admin-bro": "^1.6.0-beta.5",
    "admin-bro-expressjs": "^0.4.0",
    "admin-bro-mongoose": "^0.4.0-beta.0",
```

https://admin-bro-example-app.herokuapp.com/admin/resources/Complicated/actions/list
```
PERSON AGE:
HEIGHT:
PLACE OF BIRTH:
NESTED:
THIS NESTING IS CRAZY:
```

**Comment by wojtek-krysiak** [comment]
:) - thanks for that. Now I am working on a bulk actions just after merging that to master I will fix it (probably during the weekend).

**Comment by wojtek-krysiak** [comment]
closed with admin-bro@1.6.0-beta.8
